### PR TITLE
Update ReadMe.md file

### PR DESCRIPTION
**Pull request by AbhinavMathur29pers**
Enhanced the reliability of the documentation. Nothing major edited(code, commands, etc), just small changes here and there. Please review.
Thank you.

https://github.com/AbhinavMathur29pers/pslab-python/blob/development/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,30 +24,63 @@ This repository hosts the python library for communicating with the Pocket Scien
...
 
-and run the following
+And run the following
 
     $ sudo make clean
     $ sudo make
     $ sudo make install
 
-Now you are ready with the PSLab software on your machine.
+Now you connect PSLab device to your machine, to check whether device is detected, run `$ ls -l /dev/ttyACMx`: x is 0 for the first device.
```
Comment (CloudyPadmal) [inline README.md:53]: This is not a good check

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
Lots of errors
- Inconsistency
- Wrong commands

Double check the content and resolve conflicts.

File: README.md (hunk 1 above)
Comment (AbhinavMathur29pers) [reply README.md:53]: hello there, I'm new and i am not sure if that was meant for me or not as I only tried to fix grammar and most basic errors...

Comment (CloudyPadmal) [reply README.md:53]: Hello Abhinav, what do you mean by you're not sure?

We use `lsusb` for a connection check.

**Comment by mariobehling** [comment]
Closing this as it is below minimum quality requirements. Please make a new PR with better results if you like. Thanks.

**Pull request closed without merging**